packages/wifi-signal-strength/default.nix

```diff
@@ -6,8 +6,7 @@
   wifiDevice,
   ...
 }: let
-  # Replace the IP address with "net-vm.ghaf" after DNS/DHCP module merge
-  netvm_address = "192.168.100.1";
+  netvm_address = "net-vm";
```

Comment:
any reason to define a variable here? in e.g. [nm-launcher](https://github.com/tiiuae/ghaf/pull/584/files#diff-8e5d414928ef2d58c4b50f71cb7e0ca65585d16ce036993905e32f5b44a0fd4aR30) we just use the name directly?

Reply:
Variable was already there, I suppose that was the only reason to have it.
Variable removed.